src/main/java/seedu/address/model/room/Room.java

```diff
@@ -123,18 +124,29 @@ public Room checkIn() {
     }
 
     /**
-     * Checks out the first booking of this room and its current occupant.
-     * Future features to include exporting of receipt, setting room to housekeeping status for __x__ hours.
+     * Checks out the first booking of this room.
+     * KIV: Future features to include exporting of receipt, setting room to housekeeping status for __x__ hours.
      */
     public Room checkout() {
-        if (!hasActiveOrExpiredBooking()) {
```

Comment:
Would you need to delete the corresponding method?

Would you want to catch the `NoBookingException`?

Reply:
No need to delete, but sure.

No, why would you catch it here?